README.md

```diff
@@ -50,7 +53,7 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+castle[:guests] << town[:residents].delete_at(1)
 ```
 
```

Comment:
town.delete([:residents][1])
town[:castle][:guests] = “belle”

you need to use `town` to reference its keys